src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -81,6 +88,12 @@ export default class Toolbar extends React.PureComponent<*> {
             <header className={toolbarStyles.toolbar}>
                 <nav>
                     <div className={toolbarStyles.controlsLeft}>
+                        <Button
```

Comment:
This button should only be displayed if the `onNavigationButtonClick` has been passed in as a prop to this component.